src/app/pages/signup/signup.page.html

```diff
@@ -62,7 +62,7 @@
           <ion-slide></ion-slide>
           <ion-slide></ion-slide>
         </ion-slides>
-        <ion-button (click)="submitForm()" expand="block" shape="round" color="primary" class="nxt-btn" [disabled]="(!mobile && !email) || (!email && !mobile)" padding>
+        <ion-button (click)="submitForm()" expand="block" shape="round" color="primary" class="nxt-btn" [disabled]="(!mobile && !email)" padding>
```

Comment:
Removed redundant conditional expression for disabling form submission button.

Reply:
As long as what's in place doesn't allow for both sign in methods to be submitted.